web/experimental/new_embed.dart

```diff
@@ -3,12 +3,7 @@
 // BSD-style license that can be found in the LICENSE file.
 
 import 'package:dart_pad/experimental/new_embed.dart' as new_embed;
-import 'package:logging/logging.dart';
```

Comment:
Any particular reason for losing this logging code? It surfaces some useful errors and warnings from the runtime, if memory serves.

Reply:
Can we revert this file and add a dartanalyzer ignore comment to silence the analyzer?